examples/porter-discourse/README.md

```diff
@@ -0,0 +1,62 @@
+# Discourse Example Bundle
+
+## Why Discourse?
+[Discourse] (https://www.discourse.org/about) is the perfect example bundle because it will illustrate the need for bundling your application and using Porter. Discourse is a platform for discussion with many exciting [features] (https://www.discourse.org/features). The installation and setup process for Discourse is deceptively simple at first glance. However, it is actually extremely complicated. Although there is a standalone docker image that you can run, there is still a lot of infrastructure that needs to be configured, especially if you wish to use Discourse in production and want to take advantage of all the features that are possible. Because of this, you cannot just run the docker image and do a simple install. You will have to go through the complex process in order to set up the infrastructure and customize your Discourse. In order to install, first, you need to create a cloud server and access it. Then, you need to install docker and install discourse. To get email working, you will need to set up a mail server and get the credentials. You also need a domain name. Then, you will launch a discourse set up tool and answer questions about your hostname, email address, server address, etc. After Discourse is up and running, if an update needs to be made or maintenance is needed, another process is required. Additionally, if you want more features such as single-sign-on, plugins, or encryption, those need to be configured as well. Because of all these complex steps that are needed to install Discourse and upgrade it, putting Discourse in a bundle would greatly simplify the process. It would take away the need for individuals installing Discourse to read through all the installation instructions. Users would simply need to enter their credentials and any parameters and Porter would do the work for them. Discourse would be installed with just one command.
```

Comment:
Make sure to preview your markdown files in your editor locally. That will help you proactively spot when there are formatting issues, such as a malformed markdown links like there are here.

Also if you use hard paragraph breaks every so often (I do every 80-100 characters) it is easier for reviewers to leave comments on the lines that have problems. Otherwise I can only comment on the entire paragraph and it's more difficult to suggest changes to help out. If you open your PR's files tab and try to leave a comment yourself on a single line in this paragraph, you will quickly see what I'm talking about. 😀